doc/cephfs/add-remove-mds.rst

```diff
@@ -19,7 +19,7 @@ Add a Metadata Server
 
 #. Create the authentication key, if you use CephX. ::
 
-	$ sudo ceph auth get-or-create mds.<your-mds-id> mon 'profile mds' mgr 'profile mds' mds 'allow *' osd 'allow *' > /var/lib/ceph/mds/ceph-<your-mds-id>/keying
+	$ sudo ceph auth get-or-create mds.<your-mds-id> mon 'profile mds' mgr 'profile mds' mds 'allow *' osd 'allow *' > /var/lib/ceph/mds/ceph-<your-mds-id>/keyring
 
```

Comment:
Isn't it `/var/lib/ceph/mds/mds.<your-mds-id>` to align with the rest of the document?

Reply:
Yes,for example: the way adding OSDs in  doc /doc/install/manual-freebsd-deployment.rst.

Reply:
Yes,for example: the way adding OSDs in  doc /doc/install/manual-freebsd-deployment.rst.

Reply:
reference:  http://docs.ceph.com/docs/master/rados/configuration/auth-config-ref/#enabling-cephx
`<your-mds-id>`  change to ` {$id}`

Reply:
@renhwztetecs Yes I agree your point. But it should be `mds.{$id}` here instead of `ceph-{$id}`, in-order to match the other commands in this doc, right?

Otherwise, Am I missing something here?

Reply:
Whether we are deploying with ceph-deploy or Salt, the directory after deployment is `ceph-{$id}`, not `mds.{$id}` format.  In addition, we can also see this link(http://docs.ceph.com/docs/master/rados/configuration/auth-config-ref/#enabling-cephx), mon, mds and osd are all `ceph-{$id}` format. as follows:
`ceph auth get-or-create mgr.{$id} mon 'allow profile mgr' mds 'allow *' osd 'allow *' -o /var/lib/ceph/mgr/ceph-{$id}/keyring`

`ceph auth get-or-create osd.{$id} mon 'allow rwx' osd 'allow *' -o /var/lib/ceph/osd/ceph-{$id}/keyring`

`ceph auth get-or-create mds.{$id} mon 'allow rwx' osd 'allow *' mds 'allow *' mgr 'allow profile mds' -o /var/lib/ceph/mds/ceph-{$id}/keyring`

@joscollin 
what other commands mean?

Reply:
@renhwztetecs I meant this: [Remove a Metadata Server](https://github.com/ceph/ceph/pull/23836/files#diff-495b3d06e956ce16ebd498255ce0f5faR49).

Reply:
@sepia-liu 
yes need to be modified here.

Reply:
@renhwztetecs @joscollin Thanks. I have modified the mds path from ceph.<your-mds-id> to ceph-{$id}.